### PR TITLE
OLS-419 Conversation ID checks in REST API tests

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Conversation ID checks in REST API tests: better error description is expected

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] e2e tests

## Related Tickets & Documents

- Related Issue #[OLS-419](https://issues.redhat.com//browse/OLS-419)
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: tests/e2e/test_api.py [hunk 1]
```diff
@@ -185,6 +185,56 @@ def test_query_call_with_improper_payload():
...
+        )
+        assert response.status_code == requests.codes.ok
+        json_response = response.json()
+
+        # new conversation ID should be returned
+        assert (
+            "conversation_id" in json_response
+        ), "New conversation ID was not generated"
+        assert suid.check_suid(json_response["conversation_id"]), (
+            "Conversation ID is not it UUID format" ""
```
Comment (bparees) [inline tests/e2e/test_api.py:234]:
```suggestion
            "Conversation ID is not in UUID format" ""
```

File: tests/e2e/test_api.py [hunk 2]
```diff
@@ -185,6 +185,56 @@ def test_query_call_with_improper_payload():
...
+        expected_response = {
+            "detail": {
+                "response": "Error retrieving conversation history",
+                "cause": "Invalid conversation ID not-uuid",
+            }
+        }
+        assert json_response == expected_response
+
+
+@pytest.mark.rag
```
Comment (bparees) [inline tests/e2e/test_api.py:212]: why is this marked w/ rag?  it doesn't seem dependent on rag

File: tests/e2e/test_api.py (hunk 1 above)
Comment (tisnik) [reply tests/e2e/test_api.py]: fixed

File: tests/e2e/test_api.py (hunk 2 above)
Comment (tisnik) [reply tests/e2e/test_api.py]: ok I changed the param for response checker and removed the mark. Let's see on CI

Comment (tisnik) [reply tests/e2e/test_api.py]: we are green @bparees

**Pull request merged**